ipengine_config.py

```diff
@@ -407,10 +393,10 @@
 # Configurable for MPI initialization
 
```

Comment:
spirit is still on IPython 2.3, but I plan  to get most  of the environments upgraded by the end of the week to 3.1.0.  That startup/ code  was definitely running when I ran ipython with that profile, because I had  print statements in there, but I couldn't every find the  output from the engines.